feature/home/src/main/kotlin/nl/q42/template/home/main/ui/HomeContent.kt

```diff
@@ -36,21 +41,32 @@ internal fun HomeContent(
         viewState.userEmailTitle?.get()?.let { Text(text = it) }
 
         if (viewState.isLoading) CircularProgressIndicator()
-        if (viewState.showError) Text(text = "Error")
+        if (viewState.showError) Text(
+            text = "Error",
+            style = AppTheme.typography.body,
+            color = AppTheme.colors.error
```

Comment:
if we want to have this as a showcase screen on how to do things, the style & color here should be abstracted into a themed text component that directly use as the regular material `Text` (similar to the buttons)